Prepare for release (#332)
sentinel-hub/eo-grow: CHANGELOG.md

## [Version 1.7.7] - 2024-03-11

> - When Pipelines raise an error, the try to save the stack-trace to the logs folder into `failure.log`.

Reviewer: ```suggestion
- When a Pipeline raises an error it now saves the stack-trace to the `failure.log` file in the logs folder.
```
